vault/testing.go

```diff
@@ -2148,7 +2148,7 @@ func (m *mockBuiltinRegistry) Get(name string, pluginType consts.PluginType) (fu
 		return nil, false
 	}
 	if name == "postgresql-database-plugin" {
-		return dbPostgres.New, true
+		return toFunc(dbPostgres.New), true
```

Comment:
I don't think this wrapping is necessary, you could `return dbPostgres.New, true`

Reply:
resolved in: https://github.com/hashicorp/vault/pull/13734/commits/10b2da3c1691b8c36418824b9cb75d9c9de82dc5

Reply:
addressed in: https://github.com/hashicorp/vault/pull/13734/commits/10b2da3c1691b8c36418824b9cb75d9c9de82dc5